docs/Guide.TypeScriptIntegrationJest.md

```diff
@@ -0,0 +1,113 @@
+---
+id: Guide.TypeScriptJest
+title: Integrating TypeScript with Detox + Jest
+---
+
+## Usage
+
+### 0. Initialize Detox in your project
+
+Follow the [Getting Started](Introduction.GettingStarted.md) Guide to set up Detox with [Jest as your test runner](Guide.Jest.md) (optionally, set up Detox to [test Android](Introduction.Android.md) as well). When you are finished with that, we will make some changes to your project:
+
+- Remove Detox from `package.json` at the root level
+
+- Create a `package.json` in the `e2e` folder and add Detox and Jest to the project dependencies:
+
+```sh
+npm uninstall detox
+cd e2e
+npm init -y
+npm install -D detox jest
+```
+
+You should then move the settings for Detox from the root-level `package.json` to the `e2e/package.json`, but make sure that you update the `build` and `binaryPath` values for your Detox configurations:
```

Comment:
Just wondering - what is the value in moving your detox config to the e2e package?

Reply:
Ah I see, you are working around the type collision

Reply:
Might be good to add that as a justification for why you need to move stuff to e2e/

Reply:
👍 I added a sentence in the first section that briefly explains it.